### ISSUE TITLE
tools - drop rdmc support (Trac #1493)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1485]
Drop rdmc support as discussed on slack

```text
​[5:40 PM] Chris Weaver: Do we actually have anything which still uses RDMC?
​[5:41 PM] At a glance through combo, the answer appears to be no.
​[5:41 PM] don la dieu: lilliput
​[5:41 PM] and its optional
​[5:41 PM] Chris Weaver: Ah, so I see.
​[5:42 PM] Okay, then.
​[5:42 PM] don la dieu: olivas want's to kill it
​[5:43 PM] but a lilliput expert will need to do that
​[5:43 PM] Chris Weaver: After looking at it, I agree with his desire.
​[5:43 PM] But, yes, shooting first and asking questions later is not good.
​[5:45 PM] Alex Olivas: trying to remember how lilliput uses rdmc
​[5:45 PM] Chris Weaver: It's used for a test likelihood.
​[5:45 PM] I guess at one point it was important to check that ipdf agreed with rdmc.
​[5:55 PM] Alex Olivas: there's a pandel implementation in rdmc, right?
​[5:56 PM] so it's not just for ReaDing Monte Carlo...there's extra stuff?
​[5:57 PM] ...right
​[5:57 PM] i think we're confident at this point that they agree
​[5:58 PM] if we want to keep I3GulliverRdmcPandel then maybe we can just move the pandel impl to gulliver or lilliput or wherever's most appropriate
​[6:01 PM] Alex Olivas: added a Plain Text snippet 
* This rdmc-based class is FOR TESTING & REFERENCE PURPOSES ONLY!
31	 * It calls directly the Pandel functions from the rdmc library.
32	 * Since these functions use global variables for iceparameters etc.,
33	 * you can have only one instantiation of this class. So no good for
34	 * serious production purposes.
Add Comment Click to expand inline 6 lines
​[6:01 PM] Alex Olivas: i think we can safely kill it
​[6:11 PM] don la dieu: ok, then everything should go at once. the code, lilliput's cmakelists.txt, the tool detection
​[6:11 PM] (well, that's actually 2 commits)
```

Migrated from https://code.icecube.wisc.edu/ticket/1493
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "Drop rdmc support as discussed on slack\n\n{{{\n\u200b[5:40 PM] Chris Weaver: Do we actually have anything which still uses RDMC?\n\u200b[5:41 PM] At a glance through combo, the answer appears to be no.\n\u200b[5:41 PM] don la dieu: lilliput\n\u200b[5:41 PM] and its optional\n\u200b[5:41 PM] Chris Weaver: Ah, so I see.\n\u200b[5:42 PM] Okay, then.\n\u200b[5:42 PM] don la dieu: olivas want's to kill it\n\u200b[5:43 PM] but a lilliput expert will need to do that\n\u200b[5:43 PM] Chris Weaver: After looking at it, I agree with his desire.\n\u200b[5:43 PM] But, yes, shooting first and asking questions later is not good.\n\u200b[5:45 PM] Alex Olivas: trying to remember how lilliput uses rdmc\n\u200b[5:45 PM] Chris Weaver: It's used for a test likelihood.\n\u200b[5:45 PM] I guess at one point it was important to check that ipdf agreed with rdmc.\n\u200b[5:55 PM] Alex Olivas: there's a pandel implementation in rdmc, right?\n\u200b[5:56 PM] so it's not just for ReaDing Monte Carlo...there's extra stuff?\n\u200b[5:57 PM] ...right\n\u200b[5:57 PM] i think we're confident at this point that they agree\n\u200b[5:58 PM] if we want to keep I3GulliverRdmcPandel then maybe we can just move the pandel impl to gulliver or lilliput or wherever's most appropriate\n\u200b[6:01 PM] Alex Olivas: added a Plain Text snippet \n* This rdmc-based class is FOR TESTING & REFERENCE PURPOSES ONLY!\n31\t * It calls directly the Pandel functions from the rdmc library.\n32\t * Since these functions use global variables for iceparameters etc.,\n33\t * you can have only one instantiation of this class. So no good for\n34\t * serious production purposes.\nAdd Comment Click to expand inline 6 lines\n\u200b[6:01 PM] Alex Olivas: i think we can safely kill it\n\u200b[6:11 PM] don la dieu: ok, then everything should go at once. the code, lilliput's cmakelists.txt, the tool detection\n\u200b[6:11 PM] (well, that's actually 2 commits)\n}}}", 
    "reporter": "nega", 
    "cc": "cweaver, olivas", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "tools/ports", 
    "summary": "tools - drop rdmc support", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-12-14T23:52:21", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160215T18:14:59: 
* kjmeagher changed owner from "" to "kjmeagher"
* kjmeagher commented:

> In changeset 2406:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2406"
> remove all rdmc fixes #1485
> ```


* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160215T18:15:56: kjmeagher commented:

> removed from lilliput in r141876/IceCube

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:08:14: nwhitehorn commented:

> In changeset 170180:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170180"
> remove all rdmc fixes #1485
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
